R/render.r

```diff
@@ -422,7 +422,24 @@ data_deps <- function(pkg, depth) {
       )
     )
   )
-  deps <- bslib::bs_theme_dependencies(bs_theme)
+
+  # map secondary to component-active-bg
```

Comment:
* I can't add the mapping directly via bslib+sass unless I missed something obvious cf https://github.com/r-lib/pkgdown/issues/1678#issuecomment-845153696

* At this stage I also need to make sure there was no user value for component-active-bg.